src/main/java/org/apache/sling/distribution/journal/Reset.java

```diff
@@ -18,4 +18,5 @@
  */
 package org.apache.sling.distribution.journal;
 
+// Do not change the enum names. Implementation depends on the exact names.
```

Comment:
Some more javadoc would be appropriate here as well.

Reply:
done